conn/bind_std.go

```diff
@@ -54,23 +58,12 @@ func NewStdNetBind() Bind {
 			},
 		},
 
-		ipv4MsgsPool: sync.Pool{
-			New: func() any {
-				msgs := make([]ipv4.Message, IdealBatchSize)
-				for i := range msgs {
-					msgs[i].Buffers = make(net.Buffers, 1)
-					msgs[i].OOB = make([]byte, srcControlSize)
-				}
-				return &msgs
-			},
-		},
-
-		ipv6MsgsPool: sync.Pool{
+		msgsPool: sync.Pool{
 			New: func() any {
 				msgs := make([]ipv6.Message, IdealBatchSize)
```

Comment:
It might be worth documenting with a comment that ipv6.Message is actually an alias for internal/socket.Message, and so is ipv4, so it's the same thing.

(Also, should x/net make socket.Message public, and then we don't have to have the irrelevant `ipv6.` prefix on it?)